packages/errors/index.js

```diff
@@ -61,18 +60,23 @@ const assert = (value, optDetails, optErrorContructor) =>
   globalAssert(value, optDetails, optErrorContructor);
 Object.assign(assert, assertions);
 
+// The Agoric chain's bootstrap vat runs with a version of SES that predates
+// the addition of the 'bare' method, so we must fall back to quote behavior
+// for that environment. (2024)
```

Comment:
Is this 2024 a year reference?

Reply:
To clarify, we are planning on replacing the environment in which core evals run, and retire the bootstrap vat, at which point old agoric chains will be able to evaluate core evals without the need for this backwards compatibility. Technically replacement is not an upgrade (something something ship of Theseus)